base-focal/clortho-get

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/ruby
```

Comment:
These poor clortho scripts are duplicated in like half a dozen places already.  I wish we could DRY these out . . .